tests/test_scripts/test_packages_exporter.py

```diff
@@ -0,0 +1,24 @@
+import os
+import tempfile
+
+import pytest
+from syncer import sync
+
+from scripts.packages_exporter import Exporter
+
+IN_GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") is not None
+
+
+@pytest.mark.skipif(
+    IN_GITHUB_ACTIONS, reason="Test doesn't work in Github Actions."
+)
+def test_repomd_signer():
+    exporter = Exporter(
+        pulp_client=None, repodata_cache_dir='~/.cache/pulp_exporter'
+    )
+    db_keys = sync(exporter.get_sign_keys())
```

Comment:
I'm certain that you're trying to get sign keys from the real web server, because you don't add any sign keys or don't mock that part of the code
also why don't you use async test instead?